### ISSUE TITLE
Evaluate Transaction Clears the log panel

### ISSUE THREAD

**Issue by rthatcher** [IBM-Blockchain/blockchain-vscode-extension#720]
## Description
When using `Submit Transaction` I see the logs scroll and the result of the transaction at the bottom of the output.  However, when I `Evaluate` the same transaction the log panel clears.

I can restore the log panel by switching to Blockchain log, then back to Local Fabric log.

## Repeatable
Package/Install/Instantiate Fabcar
- Evaluate queryCar - log good
- Submit queryCar - log good
- Evaluate queryCar - **log clears**
Same behaviour using the supplied Admin identity, or a newly created Identity

## Your Environment
* IBM Blockchain Platform Extension Version: 0.3.2
* Visual Studio Code version: 1.31.0
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1): Node v8.12
* Operating System and version: Ubuntu 18.04

**Comment by heatherlp** [comment]
Thanks @rthatcher - can you let us know if you see the same behaviour on a single contract?

**Comment by rthatcher** [comment]
Yes the simple Fabcar contract was the repeatable test.  (as well as the dual Smart Contract)

**Comment by SamRasha** [comment]
Unable to recreate issue on Mac, believe it could be related to the OS used, but most likely to do with the switching between logs (when doing evaluate and submit tx's, we see the logs constantly switch between the local fabric and blockchain logs). Perhaps we could try removing this switch as it may not be necessary to see the local fabric logs as we are more interested in the blockchain ones? @cazfletch

**Comment by cazfletch** [comment]
its the local fabric logs that they will be interested in as that will tell them what happened with the transaction. @rthatcher do you still see this?

**Comment by rthatcher** [comment]
Yes still seeing it in the latest version. 
Demonstrated to @SamRasha and provided a copy of the contract I was using.

**Comment by heatherlp** [comment]
Neither me or @SamRasha can reliably recreate this, but Rob can - hopefully he can add some extra findings once he has them :)

**Comment by rthatcher** [comment]
Additional findings: @heatherlp 

The "empty log" panel being displayed is "Local Fabric", if I switch to Blockchain it is also blank, after that if I switch to Local Fabric it displays the expected Log, and then Blockchain does too.

The "empty log" panel only occurs in the **same VSCode session in which I instantiate** the contract.  
 - if I exit VSCode and restart it I do not see the problem
 - if I stop the local fabric, exit VSCode, restart VSCode then start the Fabric, I do not see the problem

The problem can be seen with the template Javascript contract using transaction1 and transaction2.
I can even switch between the 2 transactions in the Evaluate-Submit-Evaluate cycle, and still get the empty log.

The problem did not occur with a contract upgrade - only instantiate.

The problem occurs if I start with a new Project opened in the explorer view, or if I just start with a pre-packaged 'cds' contract.

The problem did not occur when I instantiated a TypeScript contract - however I then instantiated a JavaScript contract and the problem re-occured for the JavaScript contract - AND then occured for the TypeScript contract.
